README.md

```diff
@@ -716,6 +716,38 @@ Toggle a todo completion:
 response = client.todos.toggle_completion(todo_id)
 ```
 
+### [Webhook Resource](https://www.fakturoid.cz/api/v3/webhooks)
+
+Get a list of all webhooks:
+
+```ruby
+response = client.webhooks.all
+```
+
+Get a single webhook:
+
+```ruby
+response = client.webhooks.find(webhook_id)
+```
+
+Create a webhook:
+
+```ruby
+response = client.webhooks.create(url: "https://example.com/webhook", events: %w[invoice_created])
```

Comment:
Actually you should use `webhook_url` instead of `url`. I struggled with this also during implementation of Webhook model in backend.

As we return in response `url` attribute, which contains URL to Fakturoid API. Thus, in context of webooks in API, I don't feel confident using `url` name and I would prefer `webhook_url` to be consistent with response.

Reply:
@arthurwozniak I was following this https://www.fakturoid.cz/api/v3/webhooks#create-webhook is it incorrect then?

Reply:
Ok I changed it to `webhook_url` since both of them work and this one is less confusing.